core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.html

```diff
@@ -1,324 +1,646 @@
+                                    <div class="icon-icon-api" [ngClass]="{'selected': typeOfService == 'api'}"></div>
+                                    <p>API</p>
+                                </div>
+                            </section>
+                            <section class="pad-left0 service-box-section">
+                                <div tabindex="0" class="service-box" [ngClass]="{'service-box-selected': typeOfService == 'function'}" (keyup.enter)="changeServiceType('function')" (click)="changeServiceType('function')">
+                                    <div class="icon-icon-function" [ngClass]="{'selected': typeOfService == 'function'}"></div>
+                                    <p>LAMBDA</p>
                                 </div>
-                            <p>AWS</p>
+                            </section>
+                            <section class="ad-left0 service-box-section">
+                                <div tabindex="0" class="service-box" [ngClass]="{'service-box-selected': typeOfService == 'website'}" (keyup.enter)="changeServiceType('website')" (click)="changeServiceType('website')">
+                                    <div class="icon-icon-web" [ngClass]="{'selected': typeOfService == 'website'}"></div>
+                                    <p>WEBSITE</p>
+                                </div>
+                            </section>
+
+                            <section class="service-box-section align-self">OR</section>
+                            <section class="pad-left0 service-box-section">
+                                <div tabindex="0" class="service-box" [ngClass]="{'service-box-selected': typeOfService == 'custom'}" (keyup.enter)="changeServiceType('custom')" (click)="changeServiceType('custom')">
+                                    <div class="icon-icon-custom" [ngClass]="{'selected': typeOfService == 'custom'}">
+                                        <img *ngIf="typeOfService == 'custom'" src="./../../../assets/images/icons/Build-custom.svg"/>
+                                        <img *ngIf="typeOfService != 'custom'" src="./../../../assets/images/icons/Build-custom-gray.svg"/>
+                                    </div>
+                                    <p>Build Custom</p>
+                                </div>
+                            </section>
                         </div>
-                    </section>
-                    <section class="col-lg-3 col-md-3 col-sm-3 col-xs-3 pad-left0 service-box-section">
-                        <div tabindex="-1" class="service-box disabled-box" [ngClass]="{'service-box-selected': typeOfPlatform == 'azure'}" (click)="changePlatformType('azure')">
-                            <div class="icon-icon-azure"></div>
-                            <p>Azure</p>
+                        <!-- <div class="justify-center service-type-width">
```

Comment:
Remove commented code (here and other places/files)

Reply:
removed